src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/ProxyRoutes.scala

```diff
@@ -49,25 +50,42 @@ trait ProxyRoutes extends UserInfoDirectives with CorsSupport with CookieHelper
               (logRequestResultForMetrics(userInfo)) {
                 // Proxy logic handled by the ProxyService class
                 // Note ProxyService calls the LeoAuthProvider internally
-                path("api" / "localize") { // route for custom Jupyter server extension
-                  complete {
-                    proxyService.proxyLocalize(userInfo, googleProject, clusterName, request)
-                  }
-                } ~
-                  complete {
-                    proxyService.proxyNotebook(userInfo, googleProject, clusterName, request)
+                complete {
+                  proxyService.proxyRequest(userInfo, googleProject, clusterName, request)
+                }
+              }
+            }
+        } ~
+          // No need to lookup the user or consult the auth provider for this endpoint
+          path("invalidateToken") {
+            get {
+              extractToken { token =>
+                complete {
+                  proxyService.invalidateAccessToken(token).map { _ =>
+                    logger.debug(s"Invalidated access token $token")
+                    StatusCodes.OK
                   }
+                }
               }
             }
-      } ~
-        // No need to lookup the user or consult the auth provider for this endpoint
-        path("invalidateToken") {
-          get {
-            extractToken { token =>
-              complete {
-                proxyService.invalidateAccessToken(token).map { _ =>
-                  logger.debug(s"Invalidated access token $token")
-                  StatusCodes.OK
+          }
+      }
+    }
+
+  protected val jupyterRoutes: Route =
```

Comment:
I don't see where `jupyterRoutes` is referenced. 

Also it would be good if the `../api/localize` path worked for both legacy and new proxy paths.